.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,4 @@
+name: "CodeQL Configuration"
+
+paths-ignore:
+  - 'test/**'
```

Comment:
I don't think that we should be blanket-disabling an *entire* folder like this, even if this is "only" test-code, but white-list false positives individually instead.

Also, is there a reason for mixing single and double quotes in these files?
(In the source code we use *double* quotes, so that would seem preferable here as well.)

Reply:
@Snuffleupagus I can agree with that and have removed with latest commit. I decided not to remove the configuration file as you might want to add content in the future.